tests/jvm/src/test/resources/highlevel.expect

```diff
@@ -1251,8 +1264,8 @@ Symbols:
 _root_.java. => package java
 _root_.java.nio. => package nio
 _root_.java.nio.file. => package file
-_root_.java.nio.file.LinkOption. => final javadefined object LinkOption
-_root_.java.nio.file.LinkOption.NOFOLLOW_LINKS. => final javadefined val NOFOLLOW_LINKS: LinkOption
+_root_.java.nio.file.LinkOption# => abstract final sealed javadefined class LinkOption
```

Comment:
`abstract` and `sealed` are interesting additions. `final` is pretty nuts. Looking forward to getting this sorted out in https://github.com/scalameta/scalameta/issues/1162.

Reply:
haha yes, `abstract final` makes no sense :joy: